Added section for remote wasm extensions to 1.9 announcement. (#8947)
istio/istio.io: content/en/news/releases/1.9.x/announcing-1.9/_index.md

to use this functionality. If you are using the [Envoy Filter](/docs/reference/config/networking/envoy-filter/) API today
to integrate with an external authorization system, we recommend you try this feature out and give us feedback!

## Remote fetch and load of WebAssembly (Wasm) HTTP filters (Experimental)

> While it's not ready for production, it's certainly ready for experimentation.  Now Istio can fetch [WebAssembly HTTP filters](/docs/ops/configuration/extensibility/wasm-module-distribution) from remote repositories and dynamically (re)load them without restarting the proxies in your mesh.  With this you can go inject [custom C++ code](https://github.com/istio-ecosystem/wasm-extensions/blob/master/doc/write-a-wasm-extension-with-cpp.md) into your mesh to handle uses cases that go well beyond the Istio APIs.

Reviewer: ```suggestion
Now Istio supports an experimental feature to [fetch WebAssembly module](/docs/ops/configuration/extensibility/wasm-module-distribution) from remote repositories and dynamically (re)load them without restarting the proxies in your mesh.  With this you can go inject [custom C++ code](https://github.com/istio-ecosystem/wasm-extensions/blob/master/doc/write-a-wasm-extension-with-cpp.md) into your mesh to handle uses cases that go well beyond the Istio APIs.
```

Author: changed section to ```Now Istio supports an experimental feature to [fetch WebAssembly modules](/docs/ops/configuration/extensibility/wasm-module-distribution) from remote repositories and dynamically (re)load them without restarting the proxies in your mesh.  With this you can inject [custom C++ code](https://github.com/istio-ecosystem/wasm-extensions/blob/master/doc/write-a-wasm-extension-with-cpp.md) into your mesh to handle uses cases that go well beyond the Istio APIs.```
